cmake/SetupRUClient.cmake

```diff
@@ -0,0 +1,23 @@
+include(${CMAKE_CURRENT_LIST_DIR}/Utils.cmake)
+
+function(add_ruclient_to_target TARGET_NAME)
+    if(RUCLIENT_ENABLED)
+        set_target_properties(${TARGET_NAME} PROPERTIES CXX_STANDARD 20)
+        target_compile_definitions(${TARGET_NAME} PRIVATE RUCLIENT_ENABLED)
+        target_sources(${TARGET_NAME} PRIVATE rawhash_ruclient.cpp)
```

Comment:
As I'm not sure about whether we plan to keep using `rawhash_ruclient.cpp` or stick to Python client, we might also want to frop this file altogether. Any thoughts?